### DISCUSSION TITLE
manage webhppk

### DISCUSSION THREAD

**Discussion by ah-rahimi** [def-studio/telegraph#130] [Q&A] [answered]
Hello
How can we manage different robots?
For example, each bot should be sent to a separate web hook
Now, only one public class can be created for web hook, and all bot requests go to the same class
I want my different robots to refer to separate classes or can manage it

**Comment by fabio-ivona** [comment, marked as answer]
You can refer to this discussion to achieve it:  https://github.com/def-studio/telegraph/discussions/89
